e3db/client.py

```diff
@@ -46,7 +46,8 @@ def __init__(self, config):
         self.public_signing_key = config['public_signing_key']
```

Comment:
I believe that if reading in pre-existing v1 clients that this call will fail with a missing key error of some kind. But that will be resolved if it is moved down to the config version == 2 block